json_serializable/test/default_value/default_value.g.dart

```diff
@@ -34,16 +34,12 @@ DefaultValue _$DefaultValueFromJson(Map<String, dynamic> json) => DefaultValue(
             'root': ['child']
           },
       _$enumDecodeNullable(_$GreekEnumMap, json['fieldEnum']) ?? Greek.beta,
-      constClass: json['constClass'] == null
-          ? const ConstClass('value')
-          : ConstClass.fromJson(json['constClass'] as Map<String, dynamic>),
-      valueFromConverter: json['valueFromConverter'] == null
-          ? const ConstClass('value')
-          : const ConstClassConverter()
-              .fromJson(json['valueFromConverter'] as String),
-      valueFromFunction: json['valueFromFunction'] == null
-          ? const ConstClass('value')
-          : constClassFromJson(json['valueFromFunction'] as String),
+      constClass:
```

Comment:
Revert these changes. This is a bug in `pkg:analyzer` https://github.com/dart-lang/sdk/issues/46284

Reply:
The build runner seems to alternate between these 2 options at random...

Reply:
Yup. Annoying. Need to get pkg:analyzer fixed!